app/controllers/salesforce/persons_controller.rb

```diff
@@ -6,7 +6,12 @@ class Salesforce::PersonsController < Admin::AdminController
 
   def create
     begin
-      Salesforce::Person.create!(params.require(:type), user)
+      type = params.require(:type)
+      if type == "lead"
```

Comment:
Maybe we could use a constant here for "lead" and "contact" strings? Possibly `Salesforce::Lead.OBJECT_NAME.to_lower` or something like that?

Reply:
Thank you. Now I tried a different approach.